pkg/cloudprovider/providers/gce/cloud/meta/service.go

```diff
@@ -159,6 +160,11 @@ func (i *ServiceInfo) KeyIsZonal() bool {
 	return i.keyType == Zonal
 }
 
+// KeyIsProject is true if the key represents the project resource.
+func (i *ServiceInfo) KeyIsProject() bool {
+	return i.Service == "Projects"
```

Comment:
I'm not a fan of this func, but projects needed to be treated specially. Happy to hear other ideas.

Reply:
I'm ok with this for now. Can you put a comment saying why this is not great.